### ISSUE TITLE
Change case or format for env variable true/false?

### ISSUE THREAD

**Issue by batpigandme** [r-lib/remotes#461]
I (foolishly) copy and pasted in the environmental variable to stop converting warnings to errors as-is in the docs, and, naturally got:
```r
Sys.setenv(R_REMOTES_NO_ERRORS_FROM_WARNINGS=true)
#> Error in Sys.setenv(R_REMOTES_NO_ERRORS_FROM_WARNINGS = true) : 
#>   object 'true' not found
```
Turning it into a string as in the README.md (though not in the Rmd right now), of course, works:
https://github.com/r-lib/remotes/blob/3e042a2ca90336739a5426eb7bbcecdc0187ae3c/README.md#L184

Is there a reason for _not_ having the `true` and `false` in the environmental variable examples in quotation marks? Happy to do a PR to change this.

**Comment by jimhester** [comment]
Yes, please send a PR to change this!

**Comment by batpigandme** [comment]
Closed by #462  (accidentally added a space between the `#` and issue number in the PR 😳 )
